apps/25--5-clock/client/index.jsx

```diff
@@ -257,7 +257,7 @@ class Timer extends React.Component {
         <div className='author'>
           {' '}
           Designed and Coded by <br />
-          <a href='https://goo.gl/6NNLMG' target='_blank'>
+          <a href='https://goo.gl/6NNLMG' target='_blank' rel='noreferrer'>
```

Comment:
I think we do not want to use short links (regardless of the intent). @no-stack-dub-sack if you are around, can you confirm if we should swap this with a more transparent link?

Reply:
Yes, please.

Reply:
@raisedadead long time no talk! Hope you're well, man! 

Fwiw, I have no idea whether I added these short links way back when, or if they were added by someone else along the way, but in either case, I agree that links should be more transparent.

Additionally, please feel free to remove any attributions to me like this. I don't think I can claim any credit for this anymore. These projects have been touched by so many people over the years that I'm hardly the author of any of this at all! Very cool to see how its all evolved and that this is actually still around, and being used and actively developed some 5-6 years later :-)

Reply:
Thanks, Peter.

Yes - it's been a while. Lol.

We will take care of the links. Everything else can stay. These projects have a legacy and nostalgia of their own. Thanks to early contributors like yourself, we have over 10 million users who find these resources helpful.

We obviously could not have achieved this without your help :)

BTW, we have a lot in the works too.

Thanks again.